observationtree/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -230,12 +233,27 @@ object TargetObsList {
     def render(props: Props, state: State): VdomElement = {
       val obsByTarget = props.targetsWithObs.get.obs.toList.groupBy(_.target.id)
 
+      val targets   = props.targetsWithObs.get.targets.toList
+      val targetIds = targets.map(_.id)
+
       <.div(ExploreStyles.ObsTree)(
         UndoRegion[TargetsWithObs] { undoCtx =>
           DragDropContext(onDragEnd = onDragEnd(undoCtx.setter))(
             <.div(
-              props.targetsWithObs.get.targets.elements.toTagMod { target =>
-                val targetId = target.id
+              <.div(ExploreStyles.ObsTreeButtons)(
+                <.div(
+                  Button(size = Mini, compact = true, onClick = newTarget(undoCtx.setter))(
+                    Icons.New.size(Small).fitted(true)
+                  )
+                ),
+                UndoButtons(props.targetsWithObs.get, undoCtx, size = Mini)
+              ),
+              targets.toTagMod { target =>
+                val targetId     = target.id
+                val currIdx      = targetIds.indexOf(targetId)
+                val targetIdxs   = targets.zipWithIndex
+                val nextToSelect = targetIdxs.find(_._2 === currIdx + 1).map(_._1)
+                val prevToSelect = targetIdxs.find(_._2 === currIdx - 1).map(_._1)
```

Comment:
We could add functionality to `IndexedCollMod` to make this easier. I'll look into it.

Reply:
I wonder if we can use a `Zipper`?

Reply:
All this superseded by https://github.com/gemini-hlsw/explore/pull/444#issuecomment-725609876